commit db9585745c989bed9aeb5bbe6c0a5c73265ba226
Author: Ilya Tumaykin <itumaykin@gmail.com>
Date:   Fri Nov 27 16:23:24 2015 +0300

    Properly exclude all modules under test/ directory
    
    Without an asterisk only the topmost 'test' module is excluded.

diff --git a/setup.py b/setup.py
index 38ad27f..4c78cf2 100644
--- a/setup.py
+++ b/setup.py
@@ -45,7 +45,7 @@ setup(
     download_url='https://github.com/Kuniwak/vint/archive/v{version}.tar.gz'.format(version=VERSION),
     install_requires=install_requires(),
     tests_require=test_requires(),
-    packages=find_packages(exclude=['dev_tool', 'test']),
+    packages=find_packages(exclude=['dev_tool', 'test*']),
     package_data={
         'vint': [
             'asset/default_config.yaml',
